static/lang/re-en.json

```diff
@@ -3017,6 +3017,11 @@
                     "TitanicLabel": "Titanic",
                     "WyrmhideLabel": "Wyrmhide"
                 },
+                "HealersGel": {
+                    "Greater": "Greater Healer's Gel",
+                    "Lesser": "Lesser Healer's Gel",
+                    "Moderate": "Moderate Healer's Gel"
+                },
```

Comment:
> Probably just make a generic list with lesser, moderate, etc in it and use that, since this is likely an approach we'd take elsewhere too.

That is what I did at first, but reconsidered after what Kromko brought up
> In addition, in some languages, "Lesser", "Moderate" and "Greater" are written differently depending on the noun this adjective is applied to (ex., in Russian, "Lesser potion" (Малое зелье) is written differently from "Lesser gel" (Малый гель)).

Reply:
Well, to be honest, I can handle it on a per-item basis in translation.

It's not a big deal, but maybe something to consider in general regarding fragmentation of text.

Reply:
Ah. Hmm. Yeah that is a pain. I don't want to make it harder to deal with for localization than necessary, so this is fine as is I guess.